### PR TITLE
Initial app window and buttons

### PR DESCRIPTION
**Pull request by nkailasa**

### CODE REVIEW

File: GameOfLife/src/App.java [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+	private JButton pause;
+
+	public static void main(String args[]) {
+		new App();
+	}
+
+	public App() {
+		buttons = new Cell[500][500];
+
+		frame = new JFrame("Minesweeper");
```
Comment (ibeale) [inline GameOfLife/src/App.java]: Change this from minesweeper to something else

File: GameOfLife/src/App.java [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+
+import View.Cell;
+
+public class App extends JFrame {
+	private JFrame frame;
+	Cell[][] buttons = new Cell[25][25];
```
Comment (ibeale) [inline GameOfLife/src/App.java]: Currently the model is at 15x15, we will eventually put the board dimensions into a configuration class

**Review by ibeale — CHANGES_REQUESTED** [review]
Just changing the name of name of the main JFrame, otherwise, looks good and functional! Thanks Nevedita.

**Comment by nkailasa** [comment]
Creating another pull request with the required changes

**Pull request closed without merging**